README.md

```diff
@@ -570,3 +574,13 @@ nomad job status redis
 
 sudo podman ps
 ```
+
+Running the tests:
+
+```
+# Start the Podman server
+systemctl --user start podman.socket
```

Comment:
Not sure that this is necessary? the `.github/machinesetup.sh` script (which is used by both CI and vagrant provisoner) defines both the podman service and podman socket systemd units and then starts the podman service. After freshly starting up the vagrant image, the service is already running:

```
vagrant@ubuntu-impish:~/nomad-driver-podman$ sudo systemctl status podman.socket
● podman.socket - Podman API Socket
     Loaded: loaded (/etc/systemd/system/podman.socket; enabled; vendor preset: enabled)
     Active: active (listening) since Wed 2022-01-12 13:45:12 UTC; 3min 13s ago
   Triggers: ● podman.service
       Docs: man:podman-system-service(1)
     Listen: /run/podman/podman.sock (Stream)
     CGroup: /system.slice/podman.socket

Jan 12 13:45:12 ubuntu-impish systemd[1]: Listening on Podman API Socket.
```